internal/dinosql/gen.go

```diff
@@ -688,7 +688,11 @@ func New(db dbtx) *Queries {
 {{if .EmitPreparedQueries}}
 func Prepare(ctx context.Context, db dbtx) (*Queries, error) {
 	q := Queries{db: db}
-	var err error{{range .GoQueries}}
+	var err error
+	{{- if eq (len .GoQueries) 0 }}
+	_ = err
+	{{- end }}
+	{{- range .GoQueries }}
```

Comment:
With the hyphen at the front I'm pretty sure you don't need this on the same line as `var err error`.